tensorboard/uploader/proto/write_service.proto

```diff
@@ -128,3 +137,29 @@ message DeleteOwnUserRequest {
 message DeleteOwnUserResponse {
   // This is empty on purpose.
 }
+
+// Request to change the name of one experiment.
+message SetExperimentNameRequest{
```

Comment:
nit: Could you run `clang-format` on this file, please? (We don’t have a
hard lint check for it in OSS because we change these files
infrequently, but I can add one.)

Reply:
(Done in #3228.)